Testscripts/Linux/gpu-tensorflow.sh

```diff
@@ -194,11 +194,16 @@ Run_GPU_Benchmark_Test()
     if [ ! -e ${HOME}/test_results ]; then
         mkdir -p "${HOME}/test_results"
     fi
+    declare -A max_batch_size_per_mode=([inception3]=64 [vgg16]=128 [alexnet]=512 [resnet50]=64 [resnet152]=32)
 
     for mode in "${MODES[@]}"
     do
         for size in "${BATCH_SIZE[@]}"
         do
+            # Limit the batch size based on test mode to avoid the OOM(Out Of Memory) error
+            if [ $size -gt ${max_batch_size_per_mode[$mode]} ]; then
+                continue
```

Comment:
Does it mean it continues even if it hit the out of memory error?

Reply:
No, it doesn't. Replace the 'continue' with 'break' now.